src/cloud/architecture/scaled-architecture.md

```diff
@@ -0,0 +1,102 @@
+---
+group: cloud-guide
+title: Scaled architecture
+functional_areas:
+  - Cloud
+---
+
+The Cloud infrastructure scales according to your resource needs to achieve greater efficiency. The {{site.data.var.ece}} monitors your applications and can adjust capacity to maintain steady, predictable performance. Converting to this architecture helps to mitigate problems, such as latency, large spikes in traffic, or running out of storage.
+
+## Split architecture
+
+Historically, the Pro architecture consisted of 3 nodes, each containing a full tech stack. Now, the scalable infrastructure provides a tiered solution with a minimum of 6 nodes: 3 nodes for the web server and 3 nodes for the core database and other services. This split architecture provides the capability to scale tiers independently to achieve an optimal balance of performance. The core tier scales vertically (increases in size), and the web tier scales horizontally (increases instance count) and vertically (changes instance type and size).
+
+Vertical scaling must use the same instance type and size for each node:
```

Comment:
Remove `vertical`. Instance type restrictions apply based on tier and not based on type of scaling.